modules/compose/templates/docker-compose-container.yml.erb

```diff
@@ -5,6 +5,19 @@
 version: '2'
 services:
 
+########################
+# RSYSLOG
+########################
+  rsyslog:
+    image: <%= ENV["IMAGE_RSYSLOG"] %>
+    volumes:
+      - '<%= scope.lookupvar('compose::codenvy_folder') -%>/logs/haproxy/:/var/log/haproxy'
```

Comment:
it may be needed for mounting extra rsyslog configs, today we don't have any but we may have some in future